dev-docs/source/index.rst

```diff
@@ -25,6 +25,12 @@ Guides
 :doc:`GettingStarted`
    Describes the process of obtaining and building the mantid code base
 
+.. toctree::
```

Comment:
Let's leave out the index changes to avoid many merge conflicts with the other pull requests doing this.